### PR TITLE
Update the camera section

### PR DESCRIPTION
**Pull request by thartley**
this is a work in progress, but this is what i've got done today.  I put TODOs throughout for the remaining stuff that's needed.

### CODE REVIEW

File: source/includes/_api_camera.md [hunk 1]
```diff
@@ -1000,71 +858,104 @@ azimuth             | float                             | Direction that the cen
...
-### DeviceCameraInfo Attributes
+### CameraInfo Attributes
 
 Parameter           | Data Type         | Description
 ---------           | ---------------   | -----------
 bridge              | string            | GUID of bridge the camera is attached to
 camera_retention    | int               | Retention period in milliseconds
-camera_newest       | string            | Timestamp of newest event available, in EEN Timestamp format (YYYYMMDDHHMMSS.NNN)
-camera_oldest       | string            | Timestamp of oldest event available, in EEN Timestamp format (YYYYMMDDHHMMSS.NNN)
+camera_newest       | string            | Timestamp of newest event available, as [EENTimestamp](#een-types)
```
Comment (georgekarrv) [inline source/includes/_api_camera.md]: These links aren't currently working might want to at least put a todo to change them

Comment (georgekarrv) [inline source/includes/_api_camera.md]: Also it's not in the diff but there is also a DeviceSettings link in there that doesn't work maybe we make a todo to check all links?

File: source/includes/_api_camera.md [hunk 2]
```diff
@@ -1000,71 +858,104 @@ azimuth             | float                             | Direction that the cen
...
+rois | object | ROI objects, keyed by an ROI string "ID", with each value an object defined as [CameraROIs](#camerarois-attributes)
+video_transmit_mode | string | indicates when video is transmitted to the cloud
+video_capture_mode | string | indicates when video will being recorded
+video_bandwidth_factor | integer | indicates the bit rate of the video. When displaying options for this setting, you must use the data from "video_config.v.video_quality_settings.<video_resolution>.quality.<video_quality>.kbps" to show what this setting translates to for display purposes.
+video_resolution | string | indicates the resolution of the video. When displaying the options for this setting, you must use the data from "video_config.v.video_quality_settings.<video_resolution>" ("w" and "h") to show what this resolution string translates to.
+video_quality | string | indicates the quality of the video
+video_config | object | READ-ONLY object defining all the preview/video configuration parameters for each available resolution. Helps give useful information for display purposes of the "preview_resolution", "video_resolution", and "video_bandwidth_factor" settings/options.
```
Comment (jtallieu) [inline source/includes/_api_camera.md]: mostly alphabetical - at least the first characters are alpha - SEMI-PRO!

**Comment by jtallieu** [comment]
image
When I see these, it's not apparent to me what these are.  And I don't know where to go to get more info

File: source/includes/_api_camera.md [hunk 3]
```diff
@@ -973,25 +831,25 @@ timezone                      | string            | Supported timezones: If this
 guid                          | string            | guid or other physical identifier of device
 permissions                   | string            | String of one or more characters. Each character defines a permission. Permissions include: 'R' - user has access to view images and video for this camera. 'A' - user is an admin for this camera. 'S' - user can share this camera in a group share. Note: All cameras in a group must have the ‘S’ permission or the group cannot be shared
```
Comment (rmartinez-een) [inline source/includes/_api_camera.md]: 'A' - user is an admin for this camera. 'S' - user can share this camera in a group share. Note: All cameras in a group must have the ‘S’ permission or the group cannot be shared.

Should include a link to the complete list of permission letters and their respective descriptions.

Comment (georgekarrv) [inline source/includes/_api_camera.md]: Perms isn't totally fleshed out yet so maybe we just add todo to link to perms page when it's done

**Comment by devinrosen** [comment]
This is a general comment when a parameter is type object.  If that object is JSON, would love to see a visual format if the object is small enough.
ex: line 834
bridges                       | object            | Object with keys being Bridge IDs, and values being the service status of the camera on that bridge
bridges                       | object    {BridgeID(string):ServiceStatus(string)}        | Object with keys being Bridge IDs, and values being the service status of the camera on that bridge

**Comment by klaushofrichter** [comment]
@devinrosen @thartley @jtallieu  - I don't know why this shows up now in my "old PR" collection... but it can probably close?

**Comment by thartley** [comment]
Closing this, as requested by @klaushofrichter

**Pull request closed without merging**